src/components/App.js

```diff
@@ -8,20 +8,19 @@ class App extends React.Component {
   this.createNewEvent = this.createNewEvent.bind(this)
 
   this.state = {
-    urlToShare: ""
+    urlToShare: "", //populated by createNewEvent when form is submitted
+    formData: {},   //populated by form submit
   }
 
-
   }
 
 
 
 // On page load, initial user/group starter will fill out form, and on form submit, will run the following function to post to database
   createNewEvent(){
```

Comment:
pass eventData to function rather than pulling it from state createNewEvent(eventData) ... the onSubmit method I've written has to do some processing of the data stored in state to get it into the right shape to pass to the database